deliverables/README.md

```diff
@@ -31,42 +36,100 @@ What is the role of Javascript in a web page?
 
 #### CSS
 What are selectors?  
+  p { Blah } -> the p is the selector, allows developer to choose elements to change
+
 What are properties?  
+  .... -_- .... attributes of elements that you can change in CSS -> font-size, color, etc...
+
 What are values?  
+  specific property values -> color:red or color:green. 
+
 How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
+  Through nesting elements of trees.  Recieved from parent elements.
+
 What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+  Float or Margin
+
+What are the three different ways to include CSS in your project or use CSS to style a particular element?
+  Inlined, linked or embeded
 
 #### Android
-Put your html code in the `android.html` file in this folder.
+Put your html code in the `android.html` file in this folder. 
+http://htmlpreview.github.io/?https://github.com/nicholaswhite656/2015week1/blob/master/deliverables/android.html
 
 ---
 ### Section 3: Git and Github  
 What is Git?  
+  A version control system for projects
+
 What is SCM?  
+  Supply Chain Management
+
 What is a VCS?  
+  Veritas Cluster Server? The cluster software for Linux...?
+
 Why is Git useful for a developer?  
+  Easily tracks and controls changes to a project
+
 Why is Git useful for a team of developers?  
+  Exact same reason and your able to fork and clone project for multiple developers
+
 How do you create a new Git repository for a project locally?  
+  git init
+
 How do you create it on Github?  
+  Uhhh click the 'new repository' button
+
 How do you commit changes?  
+  git commit
+
 What is the difference between staging and committing changes?  
+  Commiting changes the repository file, staging is essntially a temp file for working changes before commiting.
+
 What is the difference between committing your changes and pushing them to Github?  
+  Commiting changes the repository file, pushing updates remote reference files AKA: your website/application
+
 What is the command to check the status of your current repo in git?  
-What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
-How can you look through your historical commits on the Github website?  
+  git status
+
+What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?
+  git log, git log --stat, git log -p  -> had to look up the variations
+
+How can you look through your historical commits on the Github website?
+  Use the history option/tab
+
 What is a "Merge"?  
+  Joins multiple files/development histories
+
 What is a "Pull Request"?  
+  Shows changes made to a repository
```

Comment:
A pull request is a way for you to tell someone to merge your code from a different branch or fork into their code